project/project.go

```diff
@@ -19,6 +20,8 @@ var (
 	file ini.File
 
 	pp = ""
+
+	copyToRoot = flag.Bool("r", false, "set root of the project as build out. should not be used with watch")
```

Comment:
but I really can't do anything about it here, [Watch](http://godoc.org/github.com/9fans/go/acme/Watch), [watchman](https://facebook.github.io/watchman/) and other variants are not bldy utilities so I don't have any control over how bldy is executed, and I think figuring out if bldy pid is a descendant of Watch is a bit of an overkill at this point.